frontend/src/styles/material.sass

```diff
@@ -120,3 +121,6 @@ snack-bar-container
 .mat-form-field-infix
     padding: 0
     border-top: 0
+
+.mat-menu-panel
+    margin-left: 40px
```

Comment:
```suggestion
```
We might use `xPosition="before"` property of `mat-menu` so it will be right aligned and look good independent of team name length. `margin-left` seems like not the best fit here